profile/merge.go

```diff
@@ -85,6 +86,42 @@ func Merge(srcs []*Profile) (*Profile, error) {
 	return p, nil
 }
 
+// Normalize normalizes the source profiles by multiplying each value in profiles by the
```

Comment:
I think the comment is out of date since it still talks about multiple profiles.